HW_Lesson_2/HW_Task_4.py

```diff
@@ -1 +1,5 @@
+omnivore_group = {"Harry Potter", "Jack Sparrow", "Sherlock Holmes", "James Bond"}
+vegetarian_group = {"James Bond", "Indiana Jones", "Rocky Balboa", "Han Solo"}
 
+vegetarian_group.update(omnivore_group)
```

Comment:
after this update vegetatians dict will contain omnivores which is not correct. please think how you can print both without changing original dicts